package.json

```diff
@@ -17,6 +17,8 @@
     "e2e": "playwright test src/e2e/",
     "e2e:debug": "playwright test src/e2e/ --ui",
     "e2e:smoke": "playwright test src/e2e/ --grep @smoke",
+    "e2e:ui": "playwright test src/e2e/ --grep @ui",
+    "e2e:all": "playwright test src/e2e/ --grep ''",
```

Comment:
Does this need the empty `--grep ''`?
Or, how is this different from the vanilla `e2e` target above on L17?

Reply:
completely correct, this slipped